package.json

```
@@ -1,19 +1,37 @@
 {
   "name": "open-insights",
-  "version": "1.0.0",
+  "version": "0.0.1",
   "main": "index.js",
   "repository": "git@github.com:cbsinteractive/open-insights.git",
   "license": "MIT",
   "scripts": {
     "build:dev": "webpack --config webpack.dev.js",
-    "build:dev:watch": "webpack --config webpack.dev.js --watch"
+    "build:dev:watch": "webpack --config webpack.dev.js --watch",
+    "build:doc": "rimraf doc && typedoc",
+    "lint": "eslint --fix src",
```

I find it useful to also run the TypeScript compiler `tsc` in this command to catch any obvious errors preventing compilation.
```suggestion
    "lint": "tsc --noEmit && eslint --fix src",
```